core/src/types.ts

```diff
@@ -152,3 +152,27 @@ export declare interface CollateralStatus {
     address?: string | null;
     balance?: BigNumber;
 }
+
+export type SurplusAuctionStates = 'just-started' | 'have-bids' | 'ready-for-collection' | 'requires-restart';
```

Comment:
If you export this type, `SurplusAuctionStates` should also include `collected`, I believe. Otherwise, you can maybe do it the other way around and derive it from `SurplusAuction.state`?